store-and-retrieve-data/use-storagehub-sdk/get-started.md

```diff
@@ -1,4 +1,213 @@
 ---
 title: Get Started
-description: Get up and running quickly with the StorageHub SDK.
+description: TODO
+slug: /store-and-retrieve-data/use-storagehub-sdk/get-started
 ---
+
+# Get Started with the StorageHub SDK
+
+## Overview
+
+The StorageHub SDK is a developer toolkit that makes it easy to build on DataHaven by providing direct access to StorageHub pallet functionalities for managing storage, buckets, and proofs.
+
+The SDK is split into two packages for convenient separation of concerns: `@storagehub-sdk/core` and `@storagehub-sdk/msp-client`. Both work in browser and Node.js environments. You may not always need both at once, so it’s useful to understand the purpose and typical use cases of each:
```

Comment:
It would be great to have a quick one-liner in here that distinguishes the two packages in one sentence, for example:

As a quick rule of thumb, use @storagehub-sdk/core for everything chain-facing (keys, precompiles, proofs), and @storagehub-sdk/msp-client for everything provider-facing (MSP login, buckets, file uploads/downloads).